apps/k8s-io/configmap-nginx.yaml

```
@@ -306,6 +306,7 @@ data:
           rewrite ^/sig-k8s-infra    https://github.com/kubernetes/community/tree/master/sig-k8s-infra redirect;
           rewrite ^/sig-k8s-infra-notes https://docs.google.com/document/d/1VGMyAn_Pixdg0mTwbjxRcq_-oI7xtyomG9Hoa_RpryQ redirect;
           rewrite ^/sig-k8s-infra-playlist https://www.youtube.com/playlist?list=PL69nYSiGNLP2Ghq7VW8rFbMFoHwvORuDL redirect;
+          rewrite ^/sig-release/1.33/enhancements https://github.com/orgs/kubernetes/projects/200/views/1 redirect;
```

```suggestion
          rewrite ^/sig-release/1.33/enhancements$ https://github.com/orgs/kubernetes/projects/200/views/1 redirect;
```

I made the same mistake.